mysql/server/database.sls

```diff
@@ -1,11 +1,22 @@
 {%- from "mysql/map.jinja" import server with context %}
+{%- if server.admin is defined %}
```

Comment:
Yes, much better :-) Maybe name it connection or something like that to avoid possibility of some collision.